pandas/io/excel.py

```diff
@@ -222,22 +208,20 @@
 as strings or lists of strings!
 
 >>> pd.read_excel('tmp.xlsx',
-...               na_values=['string1', 'string2'])
+...               na_values=['string1', 'string2']) # doctest: +SKIP
       Name  Value
 0      NaN      1
 1      NaN      2
 2  string3      3
 
 Comment lines in the excel input file can be skipped using the `comment` kwarg
 
->>> df = pd.DataFrame({'a': ['1', '#2'], 'b': ['2', '3']})
->>> df.to_excel('tmp.xlsx', index=False)
->>> pd.read_excel('tmp.xlsx')
+>>> pd.read_excel('tmp.xlsx') # doctest: +SKIP
     a  b
 0   1  2
 1  #2  3
 
->>> pd.read_excel('tmp.xlsx', comment='#')
+>>> pd.read_excel('tmp.xlsx', comment='#') # doctest: +SKIP
```

Comment:
I'd remove these two last examples. It assumes that the content of `tmp.xlsx` is different than before, which makes things trickier and not very easy to follow. And I don't think it's a very important use case.